logstf/logstf.sp

```diff
@@ -304,37 +330,39 @@ EndMatch(bool:endedMidgame) {
 	}
 }
 
-CacheMatchValues() {
+void CacheMatchValues() {
 	GetConVarString(g_hCvarHostname, g_sCachedHostname, sizeof(g_sCachedHostname));
 	GetConVarString(g_hCvarBlueTeamName, g_sCachedBluTeamName, sizeof(g_sCachedBluTeamName));
 	GetConVarString(g_hCvarRedTeamName, g_sCachedRedTeamName, sizeof(g_sCachedRedTeamName));
 	String_Trim(g_sCachedHostname, g_sCachedHostname, sizeof(g_sCachedHostname));
 	String_Trim(g_sCachedBluTeamName, g_sCachedBluTeamName, sizeof(g_sCachedBluTeamName));
 	String_Trim(g_sCachedRedTeamName, g_sCachedRedTeamName, sizeof(g_sCachedRedTeamName));
+	// todo: maybe work with workshop maps?
```

Comment:
I don't understand?

Reply:
Though at the end of the day... eugh. Who even uses the workshop for maps in tf2, lol. The only place I've ever seen do it was a pub server, so it really should not matter